semcore/button/src/index.d.ts

```diff
@@ -29,6 +29,10 @@ export interface IButtonProps extends IBoxProps, INeighborItemProps, IKeyboardFo
   addonLeft?: React.ElementType;
   /** Tag for the right Addon */
   addonRight?: React.ElementType;
+  /** aria-label attribute for a11y
+   * @description This prop will be required in the next major release
+   * */
+  ariaLabel?: string;
```

Comment:
As far as `ariaLabel` looks very similar to `aria-label` but still differs, we need either to name it in the isomorphic way `label` or just `aria-label` to redefine and enforce html attribute usage